nuget/lib/dependabot/nuget/update_checker/requirements_updater.rb

```diff
@@ -12,7 +12,7 @@ module Dependabot
   module Nuget
     class UpdateChecker
       class RequirementsUpdater
-        VERSION_REGEX = /[0-9a-zA-Z]+(?:\.[a-zA-Z0-9\-]+)*/.freeze
+        VERSION_REGEX = /#{Nuget::Version::VERSION_PATTERN}/.freeze
```

Comment:
Can we reference `Version::VERSION_PATTERN` directly in `initialize` or do you expect us to change it here separately from the one in `Version`?

Reply:
Ah yes you mean just reference where it's used? 👍

Reply:
Yeah, bit of a nit, but we're not changing anything about the pattern right?

Reply:
I do not think so. However not sure as I do not know much on dependabot. Only that if it works or not.

This pull request is to fix the issue where the nuget.config points to feeds that are used in the .net core projects themselves and uses packages in preview or beta, etc that dependabot manages to somehow screw up on. Hopefully this gets merged soon so that way the analyzer I use gets updated again realtime.

Reply:
Moved it inline, the only change is making it into a regex pattern whereas the `Version` pattern is a string.